scripts/windows_build_wheels.py

```diff
@@ -219,11 +219,20 @@ def fixup_wheels():
         fixup_wheel(wheel)
 
 
-def test_wheels(single_wheel=False):
+def test_wheels(python_version):
```

Comment:
For the record, the original idea was to test both cases `single_wheel==True` and `single_wheel==False` differently. That said, it can be considered as an implementation details.

Reply:
Should the first parameter be renamed from `python_version` to `python_env` ?

Reply:
>For the record, the original idea was to test both cases single_wheel==True and single_wheel==False differently. That said, it can be considered as an implementation details.

Nice, thanks for the background.

> Should the first parameter be renamed from python_version to python_env ?

Good idea. Improved.